packages/manager/modules/vps/src/dashboard/components/vps-announcement-banner/vps-announcement-banner.constant.js

```diff
@@ -0,0 +1,7 @@
+export const TASKS_INFO = {
+  DEFAULT: 'http://travaux.ovh.net/?do=details',
```

Comment:
We should refer to a specific task I think.

Reply:
Yep, i got it.